api/HTMLInputElement.json

```diff
@@ -191,47 +188,44 @@
           }
         }
       },
-      "files": {
+      "autocapitalize": {
```

Comment:
`autocapitalize` shouldn't be added: https://github.com/mdn/browser-compat-data/issues/7519